### PR TITLE
feat: #TNO-1937 - allow admins to action product requests

### PR DESCRIPTION
**Pull request by kylgarmor**
- TNO-CORE updated to 0.0.375
- admins now get an email when a user requests to subscribe or unsubscribe from a product
- no email is generated if a user cancels their request

**Editor app - Product screen with subscription request shown**
image

**Subscriber screen with various products in various states shown below:**
image

### CODE REVIEW

File: libs/net/entities/AdminConfigurableSettingNames.cs [hunk 1]
```diff
@@ -0,0 +1,15 @@
+using TNO.Core.Json;
+
+namespace TNO.Entities;
+
+/// <summary>
+/// Provides report status.
+/// </summary>
+public enum AdminConfigurableSettingNames
```
Comment (Fosol) [inline libs/net/entities/AdminConfigurableSettingNames.cs:8]: Confused how this is used?

**Comment by kylgarmor** [comment]
It’s used the same way as a string const would be. I just use a tostring to get the value to look up.
On Nov 29, 2023 at 4:40 PM -0800, Jeremy Foster ***@***.***>, wrote:

@Fosol commented on this pull request.

________________________________

In libs/net/entities/AdminConfigurableSettingNames.cs<https://github.com/bcgov/tno/pull/1287#discussion_r1410025757>:

> @@ -0,0 +1,15 @@
+using TNO.Core.Json;
+
+namespace TNO.Entities;
+
+/// <summary>
+/// Provides report status.
+/// </summary>
+public enum AdminConfigurableSettingNames


Confused how this is used?

File: libs/net/entities/AdminConfigurableSettingNames.cs (hunk 1 above)
Comment (kylgarmor) [reply libs/net/entities/AdminConfigurableSettingNames.cs:8]: It’s used the same way as a string const would be. I just use a tostring to get the value to look up.

Comment (kylgarmor) [reply libs/net/entities/AdminConfigurableSettingNames.cs:8]: Just noticed the summary is wrong from copy pasta

**Review by Fosol — APPROVED** [review]

**Pull request merged**